smol_podcaster.py

```diff
@@ -8,13 +8,21 @@
 
 import replicate
 import openai
+from openai import OpenAI
+
+client = OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
```

Comment:
Should remove one openai import here then. Why do you prefer the `client` pattern vs calling `openai` directly? Or is it because you locked the version of the sdk and it's not supported anymore?